x-pack/plugins/enterprise_search/public/applications/app_search/components/analytics/types.ts

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+interface BaseData {
+  analyticsUnavailable: boolean;
+  allTags: string[];
+  // NOTE: The API sends us back even more data than this (e.g.,
+  // startDate, endDate, currentTag, logRetentionSettings, query),
```

Comment:
I dropped these from our types - `logRetentionSettings` and `query` weren't being used at all in the frontend, and I'm no longer using `startDate`, `endDate`, `currentTag` due to changes/refactors in our analytics header (different UI elements/components for the tags & datepicker).

Reply:
Cool, that comment is helpful.

Reply:
Some day it would be nice to audit our server side API code sometime and figure out what all we actually are still using and aren't using and clean our responses up. I also might mention GraphQL unironically once or twice to give Byron hives